DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/AllDelegates/_SearchableDelegateCard.cshtml

```diff
@@ -0,0 +1,70 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.Common
+@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Delegates
+@model SearchableDelegateViewModel
+@{
+  var activeTagCss = Model.IsActive ? "nhsuk-tag" : "nhsuk-tag nhsuk-tag--grey";
+  var selfRegTagCss = "nhsuk-tag nhsuk-tag--grey";
+  var adminTagCss = "nhsuk-tag nhsuk-tag--green";
+  var passwordTagCss = Model.IsPasswordSet ? "nhsuk-tag nhsuk-tag--green" : "nhsuk-tag nhsuk-tag--red";
+}
+
+@*TODO: Search and sort functionality is part of HEEDLS-491.
+  Filename includes 'Searchable' to avoid having to change name later*@
+
+<div class="nhsuk-expander-group" id="@Model.Id-card">
+  <details class="nhsuk-details nhsuk-expander">
+    <summary class="nhsuk-details__summary">
+      <span class="nhsuk-details__summary-text" id="@Model.Id-name" name="name">
+        @Model.Name (@Model.CandidateNumber)
+      </span>
+    </summary>
+
+    <div class="nhsuk-details__text">
+      <div class="nhsuk-u-margin-bottom-4">
+        <strong class="@activeTagCss">@(Model.IsActive ? "Active" : "Inactive")</strong>
+        <strong class="@selfRegTagCss">@(Model.IsSelfReg ? "Self registered" + (Model.IsExternalReg ? " (External)" : "") : "Registered by centre")</strong>
+        <strong class="@passwordTagCss">@(Model.IsPasswordSet ? "Password set" : "Password not set")</strong>

```

Comment:
I wonder if this logic could be done on the model so we can unit test it (especially the registration status tag name)